### PR TITLE
Wired Map 

### PR DESCRIPTION
**Pull request by olliy1or**
I made this map in 5 hours, so I really want this map to be out there on the ctf server, if it didn't it will be a waste of my time if I need to improve anything let me know. I will be adding more stuff after.
screenshot_20191002_102119

### CODE REVIEW

**Comment by MinetestSam** [comment]
You seem to have add the same files twice. Please also provide a screenshot of the map

File: ctf_8.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+team.2.color = blue
+team.2.pos = (104,-42.5,-103.5)
+team.1.pos = (-105,-38.5,105.5)
+team.1.color = red
+team.1 = red
+author = olliy
+rotation = x
+h = 211
+team.2 = blue
+name = TwoTeamsTwoFlags
```
Comment (ClobberXD) [inline ctf_8.conf]: The most creative name ever, lol :P

Comment (olliy1or) [reply ctf_8.conf]: thanks :)

**Comment by olliy1or** [comment]
eh batter?

**Comment by olliy1or** [comment]
WoOps I have send .conf of another map too

**Comment by olliy1or** [comment]
you can put both of them in there :D

**Comment by Thomas--S** [comment]
License information seems to be missing

**Comment by olliy1or** [comment]
Thomas-S what license

**Comment by Thomas--S** [comment]
You haven't stated under which license you published these maps. We can only accept maps that are licensed under an appropriate license.

**Comment by ClobberXD** [comment]
> you can put both of them in there :D

No can do. Each PR should have only one map.

Also, you have to specify a license for the map, as Thomas-S said. Software licenses provide rules on using, modifying, and sharing of the software, among others. `CC BY-SA 4.0` is a very good all-rounder license, which most of our maps use. Unless you know what you're doing, I recommend choosing `CC BY-SA 4.0`. Once you've chosen your license, simply add the following line to the map's `.conf` file:

```
license = <name>
```

where `<name>` is the name of the license (e.g. `license = CC BY-SA 4.0`)

**Comment by Thomas--S** [comment]
> Unless you know what you're doing, I recommend choosing CC BY-SA 4.0

But please be aware what choosing this license means for you. Don't just use it without having read any information about it only because someone recommended it to you.

You might find this site interesting: https://choosealicense.com/licenses/cc-by-sa-4.0/

**Comment by MinetestSam** [comment]
If you don't have much time in your hands then I suggest reading this to get a better info on license https://forum.minetest.net/viewtopic.php?f=3&t=21158

**Comment by olliy1or** [comment]
working on it

**Comment by olliy1or** [comment]
how about now?

**Comment by Cahrs** [comment]
If im going to be honest, i dont think gameplay on this map is going to be that fun and/or interesting.
both sides are perfectly symmetrical, without much justification, there just isnt much todo on either side, and everyone will be funneled down the same hallways, the terrain especially seemingly made with a few WorldEdit calls.

File: Wired_Map.conf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+team.1.color = red
+team.1 = red
+author = olliy
+rotation = x
+h = 234
+team.2 = blue
+name = Wired Map 
+initial_stuff = default:pick_steel, default:sword_stone, defaut:torch 10, shooter:pistol
+r = 200
+license = CC BY SA 4.0
```
Comment (ClobberXD) [inline Wired_Map.conf]:
```suggestion
license = CC BY-SA 4.0
```

File: Wired_Map.conf [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+screenshot = ![screenshot_20191002_102119](https://user-images.githubusercontent.com/55839810/66066472-7064ef80-e4fe-11e9-8fb3-d19453c7311d.png)
```
Comment (MinetestSam) [inline Wired_Map.conf]: This needs to be changed to 
```suggestion
screenshot = Wired_Map.png
```
Also upload the screenshot and rename it to `Wired_Map.png`

**Comment by JostP** [comment]
Map size is too big... CTF crash when I start game.
R should be max 140 IIRC (or 120?)

**Comment by JostP** [comment]
And screenshot must be 3:2 pixels (for example 600 x 400 pixels)

**Comment by JostP** [comment]
hmm, how do I suggest change?
Line:
 `initial_stuff = default:pick_steel, default:sword_stone, defaut:torch 10, shooter:pistol` have to be changed into:
 `initial_stuff = default:pick_steel,default:sword_stone,defaut:torch 10,shooter:pistol` (without spaces)

File: wired_map.conf [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+team.2.pos = (2,-59,-58.5)
+team.1.pos = (2,-59,59.5)
+team.1.color = red
+team.1 = red
+author = olliy
+rotation = x
+h = 234
+team.2 = blue
+name = Wired Map 
+initial_stuff = default:pick_steel, default:sword_stone, defaut:torch 10, shooter:pistol
```
Comment (JostP) [inline wired_map.conf]:
```suggestion
initial_stuff = default:pick_steel,default:sword_stone,defaut:torch 10,shooter:pistol
```

**Comment by olliy1or** [comment]
> If im going to be honest, i dont think gameplay on this map is going to be that fun and/or interesting.
> both sides are perfectly symmetrical, without much justification, there just isnt much todo on either side, and everyone will be funneled down the same hallways, the terrain especially seemingly made with a few WorldEdit calls.

-- I know I will bring changes to this map

**Comment by ClobberXD** [comment]
No activity for 2 months, will close. Can be re-opened upon activity.

**Pull request closed without merging**